setup.py

```diff
@@ -72,6 +72,7 @@ def get_winver():
             # http://www.mingw.org/wiki/Use_more_recent_defined_functions
             ('_WIN32_WINNT', get_winver()),
             ('_AVAIL_WINVER_', get_winver()),
+            ('_CRT_SECURE_NO_WARNINGS', None),
```

Comment:
I'm not sure this is wise. If I comment this line the compiler says:

    psutil/arch/windows/security.c(89) : warning C4996: 'strcpy': This function or variable may be unsafe. Consider sing strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.

Why not use strcpy_s instead?